Source/ASVideoPlayerNode.mm

```diff
@@ -644,9 +671,11 @@ - (void)showSpinner
     }];
     
     _spinnerNode.style.preferredSize = CGSizeMake(44.0, 44.0);
-
-    [self addSubnode:_spinnerNode];
```

Comment:
The spinner can be remove and added on different threads we may should grab the `_spinnerNode` before unlocking and adding it.